src/store/proposals/proposals.selectors.ts

```diff
@@ -113,6 +118,17 @@ export const votesRatioSelector = createSelector(
   }
 );
 
+export const isGuardianSelector = createSelector(
+  [proposalDetailsSelector, accountSelector],
+  (proposal, account) => {
+    if (!proposal.data || !account) {
+      return undefined;
```

Comment:
I'm wondering if we can return false here and use `isGuardian` as boolean and don't need `!!isGuardian`

Reply:
It would make total sense in this context , but I'm afraid, if it wouldn't cause any confusion in case of potential usage in some other places.